### DISCUSSION TITLE
Thumbnail view limit

### DISCUSSION THREAD

**Discussion by arfbarky** [photoprism/photoprism#1370] [Q&A] [answered]
If I click on Library -> Originals, it starts showing a bunch of thumbnails and then at the bottom says "Limit reached, showing first 999 files" ... is this limit configurable?

**Comment by lastzero** [comment, marked as answer]
Not, we've set it based on our test with common browsers.

**Reply by arfbarky** [reply]
Oh, how do I see all the photos then? I'll have about 19k originals

**Reply by lastzero** [reply]
In the same folder? The limit is per folder.

**Reply by arfbarky** [reply]
Ok got it, hmm .. I will have to restructure my originals. thanks!
